variants/PX_HER0/variant.h

```diff
@@ -0,0 +1,231 @@
+#define PX_PWM_BUZZER       18
+#define PX_VBAT_MEAS_EN     19
+#define PX_PWR_HOLD         20
+#define PX_UART1_TX         21
+#define PX_UART1_RX         22
+#define PX_UART2_TX         23
+#define PX_UART2_RX         24
+#define PX_UART4_TX         25
+#define PX_UART4_RX         26
+#define PX_I2C1_SCL         27
+#define PX_I2C1_SDA         28
+#define PX_SPI1_CS          29
+#define PX_SPI1_SCK         30
+#define PX_SPI1_MOSI        31
+#define PX_SPI1_MISO        32
+#define PX_SPI2_CS_SF       33
+#define PX_SPI2_CS_LCD      34
+#define PX_SPI2_CS_SD       35
+#define PX_SPI2_SCK         36
+#define PX_SPI2_MOSI        37
+#define PX_SPI2_MISO        38
+#define PX_DAC1             39
+#define PX_ADC0             40
+#define PX_ADC1             41
+#define PX_ADC2             42
+#define PX_ADC3             43
+#define PX_ADC_VBAT         44
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        45
+
+// This must be a literal with a value less than or equal to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_FIRST        PB0
+#define NUM_ANALOG_INPUTS       5
+
+// On-board LED pin number
+#define LED_RED                 PH0
+#define LED_BUILTIN             PH0
+
+// On-board user button
+#define USER_BTN                PA15
+
+// SPI Definitions
+#define PIN_SPI_SS              PB6
+#define PIN_SPI_MOSI            PB5
+#define PIN_SPI_MISO            PB4
+#define PIN_SPI_SCK             PB3
+
+// I2C Definitions
+#define PIN_WIRE_SCL            PB8
+#define PIN_WIRE_SDA            PB9
+
+// I2C timing definitions (optional), avoid time spent to compute if defined
+// * I2C_TIMING_SM for Standard Mode (100kHz)
+// * I2C_TIMING_FM for Fast Mode (400kHz)
+// * I2C_TIMING_FMP for Fast Mode Plus (1000kHz)
+#define I2C_TIMING_SM           0x20302e37
+//#define I2C_TIMING_FM           0x00000000
+//#define I2C_TIMING_FMP          0x00000000
+
+// Timer Definitions
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#define TIMER_TONE              TIM6
+#define TIMER_SERVO             TIM7
+
+// UART Definitions
+#define SERIAL_UART_INSTANCE    1 //ex: 2 for Serial2 (USART2)
+
+// Default pin used for 'Serial' instance (ex: ST-Link)
+// Mandatory for Firmata
+#define PIN_SERIAL_RX           PA10
+#define PIN_SERIAL_TX           PA9
+
+// Optional PIN_SERIALn_RX and PIN_SERIALn_TX where 'n' is the U(S)ART number
+// Used when user instanciate a hardware Serial using its peripheral name.
+// Example: HardwareSerial mySerial(USART3);
+// will use PIN_SERIAL3_RX and PIN_SERIAL3_TX if defined.
+#define PIN_SERIAL1_TX          PA9
+#define PIN_SERIAL1_RX          PA10
+#define PIN_SERIAL2_TX          PA2
+#define PIN_SERIAL2_RX          PA3
+#define PIN_SERIAL4_TX          PC10
+#define PIN_SERIAL4_RX          PC11
+#define PIN_SERIALLP1_TX        PA2
+#define PIN_SERIALLP1_RX        PA3
+
+// SD card slot Definitions
+// SD detect signal can be defined if required
+#define SD_DETECT_PIN           PB7
```

Comment:
This define is required for the STM32SD library which use the SDIO or SDMMC interface.
As this MCU does not have this and you use an SPI card slot you could remove it:
```suggestion
```
Same for all SD commented related stuff